modules/shortcodes/vr.php

```diff
@@ -68,7 +68,12 @@ function jetpack_vr_viewer_shortcode( $atts ) {
 		return jetpack_vr_viewer_get_html( $url_params );
 	}
 
-	return '[vr] shortcode requires a data source to be given';
+	// add check for user
+	if ( current_user_can('editor') || current_user_can('administrator') ) {
```

Comment:
IMHO it's better to check specifically for `edit_posts` since an administrator might have been set by another admin to manage some options of the site but not edit posts, maybe due to privacy. Besides, we only need to make one check.